internal/sessions/state.go

```diff
@@ -15,34 +15,6 @@ var ErrMissingID = errors.New("invalid session: missing id")
 // timeNow is time.Now but pulled out as a variable for tests.
 var timeNow = time.Now
 
-// Version represents "ver" field in JWT public claims.
-//
-// The field is not specified by RFC 7519, so providers can
-// return either string or number (like okta).
-type Version string
```

Comment:
I guess this was not being used anymore?

Reply:
yes AFAIK it is not used